pkg/model/fileshareproto/model.proto

```diff
@@ -0,0 +1,126 @@
+// This source file has been modified by 2019 OpenSDS Authors.
```

Comment:
Can u explain the reason why create a new `model.proto` file rather than appending them in the existing one.